common/src/main/scala/com/gu/recipeasy/models/CuratedRecipe.scala

```diff
@@ -71,6 +71,16 @@ object CuratedRecipe {
     })
   }
 
+  def atomRangeFromOptionalBounds(maybeFrom: Option[Double], maybeTo: Option[Double]): Option[atom.Range] = {
+    for {
+      from <- maybeFrom
+      to <- maybeTo
+    } yield {
+      atom.Range((from * 100).toShort, (to * 100).toShort)
+    }
+    None
```

Comment:
Sorry. Yes.

`atom.Range((from * 100).toShort, (to * 100).toShort)` was initially commented out while I was sorting the atom problem, and initially returned `None` to compile and when I corrected the atom I forgot to remove the `None`